src/commands/applications/browseToApplication.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as nls from 'vscode-nls';
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { DaprApplication } from "../../services/daprApplicationProvider";
+import DaprApplicationNode from "../../views/applications/daprApplicationNode";
+import { getLocalizationPathForFile } from '../../util/localization';
+import { UserInput } from '../../services/userInput';
+
+const localize = nls.loadMessageBundle(getLocalizationPathForFile(__filename));
+
+async function browseToApplication(application: DaprApplication, ui: UserInput): Promise<void> {
+    if (application.appPort === undefined) {
+        throw new Error(localize('commands.applications.browseToApplication.noAppPort', 'No port is associated with the application \'{0}\'.', application.appId));
+    }
+
+    // TODO: Is this app always HTTP?
+    await ui.openExternal(`http://localhost:${application.appPort}`);
+}
+
+const createBrowseToApplicationCommand = (ui: UserInput) => (context: IActionContext, node: DaprApplicationNode | undefined): Promise<void> => {
+    if (node == undefined) {
```

Comment:
I never remember what `==` actually does, so I'd prefer this:
```suggestion
    if (!!node) {
```
Then any falsy value goes into the `if` body.

Reply:
That's a good catch!  I've been writing too much C# lately.  It should be `===` here and in a number of other, similar places (which I've now updated).  I've never liked the `!!value` pattern for some reason.

Reply:
I should see if there's a linter warning I can enable for that; I don't every think I've legitimately used `==`.

Reply:
I love the `!!syntax`! It's like yelling it but in reverse.

Reply:
There is!  And I've turned it on (and caught another issue).

[eqeqeq ES Lint Rule](https://eslint.org/docs/latest/rules/eqeqeq)